### PR TITLE
Multi language support

### PR DESCRIPTION
**Pull request by Luigi-Crisci**
This PR adds multi language support. This allows to generate fancy SYCL names while being more inclusive.
To add a new language:
- Upload a word list under `conf/languages/`
- Add the language to the language dictionary under `conf/languages/languages.js`
- Add the new dict key to the `option` list inside `index.html`

This PR also include support for Italian and Klingon languages

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,25 +3,30 @@
...
         const [outerStyle, leftStyle, rightStyle] = generateStyle();
+        const [leftName, rightName] = generateSyclName(document.querySelector("#lang-sel option:checked").value);
         document.querySelector('#sycl-name').style = outerStyle;
         document.querySelector('#sycl-name-left').textContent = leftName;
         document.querySelector('#sycl-name-left').style = leftStyle;
         document.querySelector('#sycl-name-right').textContent = rightName;
         document.querySelector('#sycl-name-right').style = rightStyle;
     }
-    renew();
+    document.querySelector('#gen-button').addEventListener('click', renew)
```
Comment (fknorr) [inline index.html]: We should still call `renew` directly so you get a SYCL name when loading the page.

**Review by fknorr — CHANGES_REQUESTED** [review]
Thank you Luigi for sacrificing your time in the name of an inclusive SYCL community!

Can I request that we stick to importing script sources through `<script>` instead of modules? This way one can still run it in a browser by opening the index.html, but with modules I get a CORS error instead (at least on Firefox).

File: index.html (hunk 1 above)
Comment (Luigi-Crisci) [reply index.html]: Good point, I forgot to add it

**Comment by Luigi-Crisci** [comment]
> Thank you Luigi for sacrificing your time in the name of an inclusive SYCL community!
> 
> Can I request that we stick to importing script sources through `<script>` instead of modules? This way one can still run it in a browser by opening the index.html, but with modules I get a CORS error instead (at least on Firefox).

I dropper js modules as they require a server, should work out-of-the-box now :)))

**Review by fknorr — APPROVED** [review]
Molto bene!

**Pull request merged**